### PR TITLE
Models creation

### PR DESCRIPTION
**Pull request by davide32123**

### CODE REVIEW

**Comment by davide32123** [comment]
@Ayasunsun @cassianotadaoyasumitsu @JupitersLB please all have a look at the models

@JupitersLB can you do the final validation and merge if everything ok?

File: app/models/basket.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Basket < ApplicationRecord
+  belongs_to :user
+  has_many :subscriptions
+  has_many :basket_items
```
Comment (JupitersLB) [inline app/models/basket.rb:4]: needs dependent destroy to delete basket items

File: app/models/basket.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Basket < ApplicationRecord
+  belongs_to :user
+  has_many :subscriptions
```
Comment (JupitersLB) [inline app/models/basket.rb:3]: Does this needs dependent destroy to delete subscriptions if basket is deleted

File: app/models/user.rb [hunk 3]
```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :baskets
```
Comment (JupitersLB) [inline app/models/user.rb:6]: Do baskets and subscriptions need to be dependent as well ?

File: app/models/basket.rb (hunk 2 above)
Comment (davide32123) [reply app/models/basket.rb:3]: Agreed, will add dependency

File: app/models/basket.rb (hunk 1 above)
Comment (davide32123) [reply app/models/basket.rb:4]: Agreed, will add dependency

File: app/models/user.rb (hunk 3 above)
Comment (davide32123) [reply app/models/user.rb:6]: Agreed, will add dependency

**Pull request merged**